### ISSUE TITLE
Can not remote-ssh by VS code when Oh-my-posh is installed on powershell

### ISSUE THREAD

**Issue by QuYue** [microsoft/vscode-remote-release#5499]
Issue Type: <b>Bug</b>

When I intsall Oh-my-posh (v3) on my powershell in my computer. The other computer can not remote this computer by remote-ssh function in Vscode. It will stop in "initial Vs code server" process.

VS Code version: Code 1.59.1 (3866c3553be8b268c8a7f8c0482c0c0177aa8bfa, 2021-08-19T11:56:46.957Z)
OS version: Windows_NT x64 10.0.19042
Restricted Mode: No

**Comment by roblourens** [comment]
Could you share the log from the Remote-SSH output channel when it fails?

**Comment by QuYue** [comment]
<img width="713" alt="屏幕截图 2021-08-24 104627" src="https://user-images.githubusercontent.com/24806078/130547747-b522c8dd-780b-4c53-997a-89c9251d3c3d.png">

**Comment by QuYue** [comment]
<img width="719" alt="屏幕截图 2021-08-24 105538" src="https://user-images.githubusercontent.com/24806078/130548598-6f920c27-f66f-4b3c-a673-cf6bceea3f39.png">

**Comment by QuYue** [comment]
If I add the command `Set-PoshPrompt -Theme D:\Others\My_Config\quyue.omp.json` to my 
$profile  (Microsoft.Powershell_profile.ps1）file in my server. The other computers can not connect to my server by Remote-ssh function.

**Comment by QuYue** [comment]
![image](https://user-images.githubusercontent.com/24806078/130549117-5a00537c-4973-41d9-b70c-c216831a122b.png)

**Comment by QuYue** [comment]
More interestingly,   the server even can not connect itself by remote -ssh.

**Comment by roblourens** [comment]
We expect the string `Windows PowerShell` to be written. We need a predictable string to wait for. If we send the script over before powershell has fully started, we can lose data.

I don't know how to fix this in general. It would be really helpful if someone can tell me what configuration exactly in Oh-My-Posh changes this. You could work around it by changing your powershell profile to output this string.

**Comment by Jack-Works** [comment]
Having this problem too when using oh-my-posh built-in theme

```powershell
Import-Module oh-my-posh;
Set-PoshPrompt -Theme cert;
```

> We expect the string Windows PowerShell to be written. We need a predictable string to wait for.

What is that string?

**Comment by Jack-Works** [comment]
Using theme:

```
PowerShell 7.1.4
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

Loading personal and system profiles took 1277ms.
 Jack@Jack-SB3   Workspace   
```

Not using theme (`-NoProfile`):
```
PowerShell 7.1.4
Copyright (c) Microsoft Corporation.

https://aka.ms/powershell
Type 'help' to get help.

PS C:\Users\Jack\Workspace>
```

I didn't see the string `Windows PowerShell` in the latter version too.

**Comment by QuYue** [comment]
Ok, thank you very much @roblourens. Now, I don't use oh-my-posh anymore.

**Comment by QuYue** [comment]
Yes, when the powershell shows "loading personal and system profiles took xx ms.“， the remote can not work.

**Comment by roblourens** [comment]
But it did work when not using the theme? Whenever I start powershell I get

```
Windows PowerShell
Copyright (C) Microsoft Corporation. All rights reserved.

Try the new cross-platform PowerShell https://aka.ms/pscore6

PS C:\Users\roblou>
```

**Comment by Jack-Works** [comment]
> But it did work when not using the theme? Whenever I start powershell I get

Ah, I'm using `pwsh` as my default SSH shell. So the output is `Powershell` instead of `Windows Powershell`

**Comment by Jack-Works** [comment]
@QuYue hi please don't close this because others are still having this problem. you can unsubscribe from this issue if you're not interested anymore.

**Comment by QuYue** [comment]
Oh, so sorry.  I misoperated.

**Comment by Hercier** [comment]
I have had this problem before and I add `echo "Windows Terminal"`  to make remote work, though the PowerShell will print that string itself. Also, the remote could work sometimes before the change is made. Seems there is something wrong with the string check.
